.github/workflows/integ-tests.yml

```diff
@@ -0,0 +1,211 @@
+name: Integ Tests
+
+# TODO set this up to gate new releases of Syne Tune
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  pull_request: # remove before merging, this is for testing
+    branches:   # remove before merging, this is for testing
+      - main    # remove before merging, this is for testing
```

Comment:
Fine for now, but I suppose we will want to limit the frequency of those (for instance weekly in addition to release) as they will be a tad expensive.

Reply:
Before merging this PR, I plan to push one more commit removing these lines. Then, this suite will run 1) on merges to `main` and 2) on-demand if someone with write access to the repository clicks the button to run it from the GitHub UI.

This way, we will know as soon as a bad commit is merged to main. Whereas if we only run it before a release, we might not catch a regression in main for a long time. I think at least until we have a consistent release schedule, it makes sense to start by running them against main and seeing how well it works for us.